clusterctl/examples/aws/machines.yaml.template

```diff
@@ -7,14 +7,17 @@ items:
         set: controlplane
     spec:
       versions:
-        kubelet: v1.11.2
-        controlPlane: v1.11.2
+        kubelet: v1.12.0
+        controlPlane: v1.12.0
       providerConfig:
         value:
           apiVersion: awsproviderconfig/v1alpha1
           kind: AWSMachineProviderConfig
           instanceType: m5.large
-          iamInstanceProfile: "control-plane.cluster-api-provider-aws.sigs.k8s.io"
+          iamInstanceProfile: 
+            filters:
+            - "control-plane.cluster-api-provider-aws.sigs.k8s.io"
```

Comment:
I saw a change that converted the values to a single string earlier today, does that matter?

Reply:
Nice catch! Nothing like committing partial unsaved changes unintentionally.